.babelrc

```diff
@@ -1,5 +1,6 @@
 {
   "presets": [
+    "env",
```

Comment:
Unfortunately it's necessary for testing, and removing it makes Jest complain:
> SyntaxError: Unexpected token import

I updated the `.babelrc` following [the example](https://github.com/zeit/next.js/blob/v3-beta/examples/with-jest/.babelrc) on the repo of Next.js, so there's should be no problems.